assets/src/components/books/detail/BookBorrowers.test.jsx

```diff
@@ -18,36 +15,29 @@ describe('BookBorrowers', () => {
 
   it('does not render anything if copies is empty', () => {
     testDefaultProps.copies = [];
-    const bookDetail = shallowBookBorrowers({ ...testDefaultProps });
+    const bookBorrowers = render(<BookBorrowers copies={testDefaultProps.copies} />);
 
-    expect(
-      bookDetail.find('.modal-book__borrowed-informations'),
-    ).toHaveLength(0);
+    expect(bookBorrowers.queryByText('Borrowed')).not.toBeInTheDocument();
   });
 
   it('does not render anything if there are only available copies', () => {
     testDefaultProps.copies = someBookWithAvailableCopies().copies;
-    const bookDetail = shallowBookBorrowers({ ...testDefaultProps });
-
-    expect(
-      bookDetail.find('.modal-book__borrowed-informations'),
-    ).toHaveLength(0);
+    const bookBorrowers = render(<BookBorrowers copies={testDefaultProps.copies} />);
+    expect(bookBorrowers.queryByText('Borrowed')).not.toBeInTheDocument();
   });
 
   it('renders information about who are the people who have a borrowed copy', () => {
-    const bookDetail = shallowBookBorrowers({ ...testDefaultProps });
-
-    expect(
-      bookDetail.find('.modal-book__borrowed-informations').find(Avatar),
-    ).toHaveLength(1);
+    const bookBorrowers = render(<BookBorrowers copies={testDefaultProps.copies} />);
+    expect(bookBorrowers.getByText('Borrowed')).toBeInTheDocument();
+    expect(bookBorrowers.getByText('2 years ago')).toBeInTheDocument();
+    expect(bookBorrowers.getByText('Some User')).toBeInTheDocument();
+    // TODO how can we check if Avatar is being rendered?
```

Comment:
I'm struggling with that :|

Reply:
If you set a `image_url` in userHelper.js (where the "mock" users are created), you could check the image is being rendered using `expect(bookBorrowers.getByRole('img')).toHaveAttribute('src', 'http://example.com/user.jpg')`